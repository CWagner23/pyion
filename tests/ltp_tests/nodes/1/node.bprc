1
a scheme ipn 'ipnfw' 'ipnadminep'

# ============================================================
# === Define endpoints
# ============================================================

# Default
a endpoint ipn:1.0 x
       
# Generic for tx/rx
a endpoint ipn:1.1 x

# Generic for tx/rx   
a endpoint ipn:1.2 x

# CFDP requires it  
a endpoint ipn:1.64 x
a endpoint ipn:1.65 x

# Default port where bpecho is listening
a endpoint ipn:1.100 x

# Default port where bpreceivefile is listening
a endpoint ipn:1.101 x

# Default port where bpsink is listening
a endpoint ipn:1.102 x

# ============================================================
# === Define outducts/inducts
# ============================================================

a protocol ltp 1400 100
a induct ltp 1 ltpcli
a outduct ltp 1 ltpclo
a outduct ltp 2 ltpclo
r 'ipnadmin node.ipnrc'
w 1
s
