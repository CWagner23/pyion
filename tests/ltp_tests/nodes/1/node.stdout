[i] admin pgm using SDR parm overrides from node.ionconfig.
wmKey:           0
wmSize:          5000000
wmAddress:       0
sdrName:        ''
sdrWmSize:       0
configFlags:     1
heapWords:       5000000
heapKey:         -1
logSize:         0
logKey:          -1
pathName:       '/tmp'
Stopping ionadmin.
Stopping ionsecadmin.
Stopping ionadmin.
Stopping ltpadmin.
Stopping ipnadmin.
Stopping bpadmin.
eeefNOTE: udplso now gets transmission data rate from the contact plan.  txbps is still accepted on the command line, for backward compatibility, but it is ignored.
gggshgeeefgggshg